src/replication/log_consumer.cpp

```diff
@@ -181,6 +191,42 @@ namespace cubreplication
 		break;
 	      }
 
+	    MVCCID mvccid = se->get_mvccid ();
+
+	    // apply only some selected mvccids until we reach last m_ack_stream_position, then continue normally
+	    if (is_filtered_apply_segment (se->get_stream_entry_end_position ()))
+	      {
+		_er_log_debug (ARG_FILE_LINE, "Filtered apply: Entered filtered apply segment, m_filtered_apply_end: %llu\n",
```

Comment:
Shouldn't group commit stream entries be skipped during filtered phase?

Reply:
I think they do get skipped